RCTComms/transport.py

```diff
@@ -401,3 +400,82 @@ def send(self, data: bytes, dest):
 
     def isOpen(self):
         return self.__socket is not None
+
+class RCTSerialTransport(RCTAbstractTransport):
+    '''
+    Serial Transport
+    socket.socket wrapped in a serial.Serial object wrapped in an abstract transport (move over, turducken)
```

Comment:
why???
line too long

Reply:
Resolved with 662a21c
Forgot I had added that while I was in the "what am I doing" stage, oops. Docstring is normal now.